lib/matplotlib/axes/_base.py

```diff
@@ -3492,10 +3492,27 @@ def set_xlabel(self, xlabel, fontdict=None, labelpad=None, *,
             loc = (loc if loc is not None
                    else mpl.rcParams['xaxis.labellocation'])
         _api.check_in_list(('left', 'center', 'right'), loc=loc)
+        x = None or kwargs.get('x')
+        ha = None or kwargs.get('ha')
```

Comment:
Why the `or`? `None` is always falsy.

Reply:
Here, x or ha will be the value passed by the user. If no value is passed, it'll default to None. This is used in the subsequent if/else blocks.

Reply:
My bad. I forgot that `kwargs.get()` returns None if the key isn't found.